src/relay/collage/utils.cc

```diff
@@ -134,6 +134,19 @@ bool MustBeLowered(const Expr& expr) {
   return false;
 }
 
+std::vector<std::string> SplitString(std::string stmt, const char* del) {
```

Comment:
Just define this in `gather_partition_specs.cc` where it's used, since it is a one-off function

Reply:
I have defined generic split string functions, which can be useful for any kind of splitting of string statement in future. So I defined as utility function.